### ISSUE TITLE
Add country level data for marginal intensity from the UNFCCC

### ISSUE THREAD

**Issue by mrchrisadams** [thegreenwebfoundation/co2.js#97]
I've just come off a call with the UNFCCC about use of the IFI marginal intensity figures they had on their website.

There is currently no provider of marginal intensity open data at present, but after a series of calls, I can now confirm that we _can_ indeed use this data and publish it. The data is linked below, but it likely needs a few small changes for it to be usable. I've outlined a few steps:

- [x] adding 2 and 3 character ISO codes - this makes lookups with other software much more reliable
- [x] add a source column linking back to the IFI figures
- [x] make a decision about including just the OM, or the wider set of CM (combined margins account for planned projects coming onstream and affecting the grid mix)

You can read more below:

https://github.com/thegreenwebfoundation/grid-intensity-go/issues/8

**Comment by fershad** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Include EMBER and UNFCCC data in CO2.js](https://trello.com/c/vcBJO5ig/69-include-ember-and-unfccc-data-in-co2js)

**Comment by fershad** [comment]
Update merged with #102. Happy to close @mrchrisadams ?

**Comment by mrchrisadams** [comment]
Hi @fershad  - I think the only thing we need left to close this is to be a explicit about the licensing terms of for the UNFCCC data.

We have the go ahead to publish this ourselves, as open data, but as part of the open data triage process we did, we said we'd make sure we added a similar disclaimer about the use of the datasets, along with some info about origin of the underlying data. We do this with some of the existing data like this:

https://www.thegreenwebfoundation.org/green-web-datasets

I can write that on the Friday in more detail so we have this ready for publishing on Monday with v0.11.x, and I can make a PR for this to add to the readme.
